README.md

```diff
@@ -1,23 +1,34 @@
-# avail-light
+<div align="Center">
+<h1>avail-light</h1>
+<h3> Data availability and computational integrity verified by avail light</h3>
+</div>
 
-[![Build status](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml/badge.svg)](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml) [![Code coverage](https://codecov.io/gh/maticnetwork/avail-light/branch/main/graph/badge.svg?token=7O2EA7QMC2)](https://codecov.io/gh/maticnetwork/avail-light)
+<br>
 
-Light client for Data Availability Blockchain of Polygon 💻
+[![Build status](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml/badge.svg)](https://github.com/maticnetwork/avail-light/actions/workflows/default.yml) [![Code coverage](https://codecov.io/gh/maticnetwork/avail-light/branch/main/graph/badge.svg?token=7O2EA7QMC2)](https://codecov.io/gh/maticnetwork/avail-light)
 
 ![demo](./img/prod_demo.png)
 
 ## Introduction
 
-Naive approach for building one DA light client, which will do following
+Build a data availability light client that will do following:
```

Comment:
I'd suggest something like: `avail-light` is a data availability light client that can do the following:

Reply:
Otherwise changes lgtm. Let's make sure we revisit all READMEs and docs on a regular basis and iteratively improve them.

Reply:
Addressed